app/src/main/AndroidManifest.xml

```diff
@@ -37,14 +37,18 @@ limitations under the License.
                 android:label="@string/app_name">
         </activity>
         <activity
-                android:name=".ui.FlashScreenActivity"
-                android:label="@string/title_activity_flash_screen">
+                android:name=".ui.tutorial.TutorialActivity"
```

Comment:
Launcher activity should be `FlashScreenActivity`.

Reply:
I have used the preference which checks if the app is first time launched or not.

Reply:
I have used the preference which checks if the app is first time launched or not.

Reply:
But first show flash screen then tutorial. And manage first time launch in flash screen.